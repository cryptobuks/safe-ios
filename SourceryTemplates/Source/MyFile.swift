//
//  Copyright Â© 2018 Gnosis Ltd. All rights reserved.
//

import Foundation
