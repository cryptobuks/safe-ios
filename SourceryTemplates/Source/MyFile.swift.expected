//
//  Copyright Â© 2018 Gnosis Ltd. All rights reserved.
//

import Foundation

public func hello() -> String {
    return "Hello!"
}
